spec/processing/mini_magick_spec.rb

```diff
@@ -23,9 +23,7 @@
   describe "#convert" do
     it "should convert from one format to another" do
       @instance.convert('png')
-      img = ::MiniMagick::Image.open(@instance.current_path)
-      expect(img['format']).to match(/PNG/)
-      expect(@instance.file.extension).to eq('png')
```

Comment:
I think we need to include the file extension check here, unless you'd also like to write a file extension matcher. :smile: